### DISCUSSION TITLE
Feedback on multiple monitor snapping

### DISCUSSION THREAD

**Discussion by akoller1993** [rxhanson/RectanglePro-Community#179] [Ideas]
Hi Ryan,
Not sure if this already exists, but is there any way to have like a pressure treshhold when snapping a window in a multi monitor setup (with a mouse, not commands)? If I try to snap a window to the left, having another monitor left to the one with the window, I have to very carefully place the cursor on the edge of the monitor to get the snap, otherwise the window just moves to the left monitor. It would be nice to have some feedback or threshhold that snaps the window first and if I keep pushing, then it goes to the other monitor. Other than that, the app is flawless, I could not conceive my life without it. Thanks a lot.

Andy

**Comment by rxhanson** [comment]
Thanks for using the app! The reason that this hasn't been added in already is that I usually just use the window throw to snap my windows, and I don't really ever drag to the screen edge to snap. I do have this on my backlog, but I don't have a timeframe for when I would get it in.

**Reply by akoller1993** [reply]
Thanks a lot!
